modules/compliance-timeout.adoc

```diff
@@ -33,5 +33,5 @@ schedule: '0 1 * * *'
 timeout: '10m0s' <1>
 maxRetryOnTimeout: 3 <2>
 ----
-<1> The `timeout` variable is defined as a duration string, such as `1h30m`. The default value is `30m`.
+<1> The `timeout` variable is defined as a duration string, such as `1h30m`. The default value is `30m`. The timeout can be disabled by setting the value to `0s`.
```

Comment:
```suggestion
<1> The `timeout` variable is defined as a duration string, such as `1h30m`. The default value is `30m`. To disable the timeout, set the value to `0s`.
```

Reply:
Yep, I agree. I like that a lot better. Thank you!